package.json

```diff
@@ -45,9 +45,9 @@
     "@bower_components/ng-file-upload-shim": "danialfarid/angular-file-upload-shim-bower#^12.0.4",
     "@bower_components/randexp": "fent/randexp.js#^0.4.0",
     "@ovh-ux/manager-config": "^0.2.0",
-    "@ovh-ux/manager-core": "^5.2.5",
-    "@ovh-ux/manager-navbar": "^0.5.0",
     "@ovh-ux/manager-server-sidebar": "^0.2.0",
+    "@ovh-ux/manager-core": "^6.0.0",
```

Comment:
Using `yarn` commands normally keeps dependencies sorted.

Probably related to a conflict resolution but could you please keep them sorted?

Thanks!